Core/Inc/analyzer.h

```diff
@@ -4,6 +4,7 @@
 //#include <nerduino.h> Replace
 #include "datastructs.h"
 #include "segment.h"
+#include "Core/Drivers/Embedded-Base/general/include/sht30.h"
```

Comment:
ive discovered that for whatever reason it doesnt love the full path, so actually just "sht30.h" will work better